cmd/sql/flow.go

```diff
@@ -503,27 +502,58 @@ func executeDeployCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	dagsPath := filepath.Join(projectDir, ".deploy/"+env+"/dags")
-	if err := os.MkdirAll(dagsPath, os.ModePerm); err != nil {
-		return fmt.Errorf("error creating directories for %v: %w", dagsPath, err)
+	dagsPath := filepath.Join(config.WorkingPath, "dags/.astro-sql-deploy/"+env)
+
+	if err := updateDags(dagsPath, args); err != nil {
+		return fmt.Errorf("error updating DAGs based on workflows: %w", err)
 	}
-	var workflowName string
-	if len(args) > 0 {
-		workflowName = args[0]
+
+	if context.IsCloudContext() {
+		err = uploadDagsToCloud(dagsPath)
+		if err != nil {
+			return err
+		}
+	} else {
```

Comment:
There may be other contexts as well; it may be worth explicitly confirming if it is software context - and raising an exception if it is neither

Reply:
@tatiana @pankajkoti at the moment `isCloudContext` is a boolean so it can only be cloud or not cloud. there's no third option we can check.